Source/External/ExCSS/StylesheetReader.cs

```diff
@@ -183,4 +183,4 @@ private void BackUnsafe()
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Please do not include this file in your PR.

Reply:
fixed